x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/hooks.test.tsx

```diff
@@ -22,15 +24,39 @@ jest.mock('../../../../common/containers/sourcerer', () => ({
   getScopeFromPath: jest.fn(),
 }));
 
-test('getAggregatableFields', () => {
-  expect(getAggregatableFields(mockBrowserFields)).toMatchSnapshot();
-});
+describe('getAggregatableFields', () => {
+  test('getAggregatableFields when useLensCompatibleFields = false', () => {
+    expect(
+      getAggregatableFields(mockBrowserFields.base?.fields as GetAggregatableFields)
+    ).toMatchSnapshot();
```

Comment:
Would you mind either adding the actual object since it's only two fields or or make it an inline snapshot? People don't often look at the actual snapshots when they're in other files